### PR TITLE
README Update

### PR DESCRIPTION
**Pull request by lxbsz**
Add logger setting and system config section.

### CODE REVIEW

**Comment by pkalever** [comment]
Looks good to me.

How about mentioning of different logdir configuration options with their priorities ?

File: README.md [hunk 1]
```diff
@@ -110,6 +110,32 @@ glfs: /volume@hostname/filename
...
+uncomment the following line in /etc/tcmu/tcmu.conf and set your level number:
+
+\# log_level = 3
+
+- System configuration:
+
+The default configuration file is installed into /etc/tcmu/tcmu.conf.
+
+The system configuration support dynamic reloading without restarting the
+runner deamon based the inotify feature. Do change the operations as need,
```
Comment (pkalever) [inline README.md]: s/deamon/daemon/ ?

Comment (lxbsz) [reply README.md]: Will fix :)

@pkalever  Thanks,

**Comment by lxbsz** [comment]
@pkalever 

Yes, I will add it now.

Thanks,

**Comment by lxbsz** [comment]
Updated, @pkalever please review.
Thanks,

File: README.md [hunk 2]
```diff
@@ -110,6 +110,41 @@ glfs: /volume@hostname/filename
...
+3. INFO
+4. DEBUG
+5. DEBUG SCSI CMD
+
+And the default logging level is 3, if you want to change the default level,
+uncomment the following line in /etc/tcmu/tcmu.conf and set your level number:
+
+\# log_level = 3
+
+The log file's location could be configurd in /etc/tcmu/tcmu.con too.
```
Comment (pkalever) [inline README.md]: s/configurd/configured/ ?
s/tcmu.con/tcmu.conf/ ?

File: README.md [hunk 3]
```diff
@@ -110,6 +110,41 @@ glfs: /volume@hostname/filename
...
+
+The priority of the logdir setting is:
+
+1. --tcmu_log_dir/-l `LOG_DIR_PATH` [Highest]
+2. export TCMU_LOGDIR="/var/log/mychoice/"
+3. /etc/tcmu/tcmu.conf
+4. default /var/log/ as hard coded [Least]
+
```
Comment (pkalever) [inline README.md]: @lxbsz  Just a cosmic change, how about:

The priority of the logdir setting can be managed via following options:
1. Cli argument
     eg: --tcmu_log_dir/-l `LOG_DIR_PATH` [Highest prio]
2. Environment variable
    eg: export TCMU_LOGDIR="/var/log/mylogdir/"
3. Configuration file
     eg: uncommenting and adjusting value of 'log_dir_path' at  /etc/tcmu/tcmu.conf
4. Default logdir as hard coded i.e. '/var/log/' [Least prio]

IMO, doesn't need a re-spin, just for this :-)

Comment (lxbsz) [reply README.md]: Looks much better than mine :-)

**Comment by lxbsz** [comment]
@pkalever  
Please review.

**Comment by lxbsz** [comment]
@pkalever Thanks.

File: README.md [hunk 4]
```diff
@@ -110,6 +110,42 @@ glfs: /volume@hostname/filename
...
+- System configuration:
+
+The default configuration file is installed into /etc/tcmu/tcmu.conf.
+
+The system configuration support dynamic reloading without restarting the
+runner daemon based the inotify feature. Do change the operations as need,
+save and close the /etc/tcmu/tcmu.conf, and then the system configuration
+reloading thread will apply the changes to the runner daemon for you.
+
```
Comment (mikechristie) [inline README.md]: This seems to detailed for users. How about just:

tcmu-runner's configuration systems supports dynamic reloading without restarting the daemon. To change values open /etc/tcmu/tcmu.conf, update the value, and then close the file.

Comment (lxbsz) [reply README.md]: Yeah. Agree.

**Comment by lxbsz** [comment]
@mikechristie 
Updated

**Comment by mikechristie** [comment]
Thanks.

**Pull request merged**